modules/trusted-firmware-m/tfm_boards/nrf54l15_cpuapp/cpuarch.cmake

```diff
@@ -1,21 +1,10 @@
-#-------------------------------------------------------------------------------
-# Copyright (c) 2024, Nordic Semiconductor ASA.
+#
+# Copyright (c) 2021 - 2024, Nordic Semiconductor ASA.
```

Comment:
We shall not use ranges in copyrights. Just use first year when a feature was added

Reply:
fixed